tests/service/services_absent.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Please use "#!/bin/bash -eu" instead. Then no entry in the ignore file will be needed.